### PR TITLE
Added PVR.EpgEventIcon guiinfo label

### PR DESCRIPTION
**Pull request by lightglitch**
## Description
Similar to #14361 but for the VideoPlayer

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by lightglitch** [comment]
I don't think so, this should work similar to the VideoPlayer.Plot API, it's not for the PVR.

**Review by ksooo — CHANGES_REQUESTED** [review]
Not Videoplayer.EpgEventIcon, but PVR.EpgEventItem. Videoplayer knows nothing about EPG.

**Comment by lightglitch** [comment]
@ksooo  but  you also have epg info (related to pvr) on videoplayer like:

VIDEOPLAYER_HAS_EPG
VIDEOPLAYER_NEXT_PLOT
VIDEOPLAYER_NEXT_STARTTIME

```
///   \table_row3{   <b>`VideoPlayer.HasEpg`</b>,
///                  \anchor VideoPlayer_HasEpg
///                  _boolean_,
///     Returns true if epg information is available for the currently playing
///     programme (PVR).
///   }
///   \table_row3{   <b>`VideoPlayer.NextPlot`</b>,
///                  \anchor VideoPlayer_NextPlot
///                  _string_,
///     Plot of the programme that will be played next (PVR).
///   \table_row3{   <b>`VideoPlayer.NextStartTime`</b>,
///                  \anchor VideoPlayer_NextStartTime
///                  _string_,
///     Start time of the programme that will be played next (PVR).
///   }
```

**Comment by ksooo** [comment]
> but you also have epg info (related to pvr) on videoplayer like

 How about trusting me a little bit? I know this code for a couple of years now. Well, all those labels are subject to get moved to PVR. They were added to VideoPlayer scope at times were we did not have a sharp eye on software architecture...

**Comment by lightglitch** [comment]
@ksooo Request changes made.

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -4675,6 +4675,11 @@ const infomap pvr[] =            {{ "isrecording",              PVR_IS_RECORDING
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`PVR.EpgEventIcon`</b>,
+///                  \anchor PVR_EpgEventIcon
+///                  _string_,
+///     Returns the thumbnail for the epg event associated with the item (if it exists)
```
Comment (ksooo) [inline xbmc/GUIInfoManager.cpp]: Please change the text to reflect that this returns the icon of the currently playing epg event, if any - like phrased in the other comments. => "Returns the icon of the currently playing epg event, if any."

File: xbmc/guilib/guiinfo/GUIInfoLabels.h [hunk 2]
```diff
@@ -593,6 +593,7 @@
 #define PVR_TIMESHIFT_PROGRESS_START_TIME (PVR_STRINGS_START + 70)
 #define PVR_TIMESHIFT_PROGRESS_END_TIME   (PVR_STRINGS_START + 71)
 #define PVR_STRINGS_END                   PVR_TIMESHIFT_PROGRESS_END_TIME
+#define PVR_EPG_EVENT_ICON                (PVR_STRINGS_START + 72)
```
Comment (ksooo) [inline xbmc/guilib/guiinfo/GUIInfoLabels.h]: Move this line above the line containing PVR_STRINGS_END, adapt that line to: `#define PVR_STRINGS_END PVR_EPG_EVENT_ICON `. You get the gist of this?

**Review by ksooo — CHANGES_REQUESTED** [review]
Only some minors.

**Comment by ksooo** [comment]
Just a comment regarding the wording: Added PVR.EpgEventIcon API

What you are doing in Kodi terminology is adding a guiinfo label, not an API.

**Comment by lightglitch** [comment]
Review requests implemented.

**Review by FernetMenta — APPROVED** [review]
What @ksooo says

**Review by ksooo — APPROVED** [review]
Nice, thanks.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
@sualfred fyi

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**